src/components/SharedComponents/PhotoCount.js

```diff
@@ -0,0 +1,58 @@
+// @flow
+
+import { View } from "components/styledComponents";
+import * as React from "react";
+import Svg, { Path, Text } from "react-native-svg";
+
+type Props = {
+  count: number,
+  size?: number,
+  shadow: boolean,
+};
+
+const PhotoCount = ( { count, size, shadow }: Props ): React.Node => {
+  if ( count <= 0 || count > 99 ) {
```

Comment:
Count should not be negative, so you don't need to account for negative values here. So, please change to `=== 0` .